packages/backend-ai/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@aws-amplify/backend-ai",
+  "version": "0.1.0",
+  "type": "module",
+  "publishConfig": {
+    "access": "public"
+  },
+  "exports": {
+    ".": {
+      "types": "./lib/index.d.ts",
+      "import": "./lib/index.js",
+      "require": "./lib/index.js"
+    }
+  },
+  "dependencies": {
+    "@aws-sdk/client-bedrock-runtime": "^3.602.0",
+    "aws-cdk-lib": "^2.132.0",
```

Comment:
+1 to chris asking if we need this at all, but if we do, it must only be a peerDependency